### ISSUE TITLE
How to deal with Long time video data which has1000 frames

### ISSUE THREAD

**Issue by dotsonliu** [lucidrains/TimeSformer-pytorch#3]
How to deal with Long time video data which has1000 frames，img size is 224*224，patch size is 16*16，then there has 196000  patchs,how to deal with it? put it into transformer once???

**Comment by tcapelle** [comment]
No, you ahve to skip frames or use less. The paper deals with this issue skipping frames. (one every 15)

**Comment by dfan** [comment]
Could also try substituting in a sparse / linear self-attention mechanism
